docs/usage/frameworks/fastapi.rst

```diff
@@ -2,7 +2,7 @@
 FastAPI Integration
 ===================
 
-Advanced Alchemy provides integration with FastAPI through its repository and service patterns.
+Advanced Alchemy's repository and service patterns work well within FastAPI applications.
```

Comment:
iiuc, there's no integration code within advanced-alchemy for FastAPI. the docs are just showing that repositories and services can work within FastAPI apps, correct?

Reply:
@sloria That's correct. As soon as I figure out what's blocking these flask CLI tests (they are only failing in the CI right now), i'll move over to the FastAPI cli.

Reply:
no urgency on my end for FastAPI cli support (not using it myself), though i'm keen to to try out litestar :) really great work on litestar and advanced-alchemy. advanced-alchemy would have be super useful on my last work project ❤️